pdns/dnsdistdist/docs/guides/tls-sessions-management.rst

```diff
@@ -0,0 +1,56 @@
+TLS Sessions Management
+=======================
+
+TLS sessions
+------------
+
+One of the most costly TLS operation if the negotiation of a new session, since both the client and the server need to generate and agree on cryptographic materials. In order to reduce that cost, TLS implements what is called session resumption, where a client opening a new connection to a server can reuse the cryptographic materials negotiated for a previous TLS session.
```

Comment:
```suggestion
One of the most costly TLS operation is the negotiation of a new session, since both the client and the server need to generate and agree on cryptographic materials. In order to reduce that cost, TLS implements what is called session resumption, where a client opening a new connection to a server can reuse the cryptographic materials negotiated for a previous TLS session.
```
Typo, if -> is